### ISSUE TITLE
Suggestion: statistics in dock

### ISSUE THREAD

**Issue by Elowan** [FailcoderAddons/supervillain-ui#186]
hi there,

it would be totally cool (at least I think so) to have some statistics displayed in dock. (like the skada, time, guild an so on). I am thinking of: "Player Deaths", "Mobs killed", "Gold spent", "time played", "you name it".

Do you see a possibility?

Kind regards,

Elo

**Comment by finalsliver** [comment]
alot of this is already boilt in. frames change for pvp to highlight things like frags and player deaths so you can see your KDR. others are tracked in the docklets which can be changed and moved at top and bottom of the screen

**Comment by Elowan** [comment]
ah, cool! seems, I need some "description of all functions and magic of SVUI" - as I am missing and overlook most stuff, I am asking for ;)

**Comment by Elowan** [comment]
ok - seems this stuff just works for PvP situations? I was refering to the Pve related statistics.

**Comment by finalsliver** [comment]
Time played each session could have some worth, player deaths & mobs killed seem a tad niche tbh in PvE. 

While it could be added it would be pretty low priority I reckon.

Such info can be found in the statistics window ingame (shift+y) and I suppose a docket could pull info from there but still sounds a tad niche.

**Comment by Elowan** [comment]
yes, maybe a bit niche.. but as a supervialin, I´d like to brag with "mobs slain" for example ;)
